Wikipedia/Code/NSUserDefaults+WMFExtensions.swift

```diff
@@ -533,7 +533,7 @@ let WMFSearchLanguageKey = "WMFSearchLanguageKey"
 
     var autoSignTalkPageDiscussions: Bool {
         get {
-            return bool(forKey: UserDefaults.Key.autoSignTalkPageDiscussions)
+            return (value(forKey: UserDefaults.Key.autoSignTalkPageDiscussions) as? Bool) ?? true
```

Comment:
Might be better to set a default in [registerDefaults](https://github.com/wikimedia/wikipedia-ios/blob/f1fbf36090dea6ddf31426337e2eac67b287ac0e/Wikipedia/Code/AppDelegate.m#L29) so you can keep the bool(forKey) here and any automatic conversion it does under the hood for you.

Reply:
done